test/integration/user_can_edit_their_info_test.rb

```diff
@@ -17,7 +17,7 @@ class UserCanEditTheirInfoTest < ActionDispatch::IntegrationTest
     assert_equal edit_user_path(user), current_path
 
     fill_in "City", with: "Gotham"
-    fill_in "State", with: "NY"
+    select('New York', :from => 'user_state')
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.